disclosure/urls.py

```diff
@@ -16,9 +16,7 @@
                                'document_root': settings.STATIC_ROOT,
                                'show_indexes': True,
                            }),
-                       url(r'', include('locality.urls')),
-                       url(r'^locations/(?P<fips_id>[0-9]+)$',
-                           views.location_view),
```

Comment:
Don't delete this yet, [it still exists](https://github.com/bcipolli/opendisclosure-backend/blob/live-search/disclosure/views.py#L34) as a stub.